File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/35_compito_java_2019_11_20/Pallamano.java


public class Pallamano extends Squadra {

    private double reti;

    public Pallamano(int codice, String sport, String nome, int vinte, int perse, double reti) {
        super(codice, sport, nome, vinte, perse);
        this.reti = reti;
    }

    public String toString() {
        return nome + "\t" + codice + "\t" + vinte + "\t" + perse + "\t-\t-\t" + reti + "\t" + sport;
    }
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/35_compito_java_20191120/Pallamano.py
from Squadra import Squadra


class Pallamano(Squadra):
    def __init__(self, codice, sport, nome, vinte, perse, goal):
        super().__init__(codice, sport, nome, vinte, perse, goal)

    def __str__(self):
        return super().__str__() + "\t-\t-\t" + str(self._goal) + "\t" + self._sport
